src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import Avatar from '../Avatar';
+import TooltipMixin from '../../mixins/tooltip';
+import ApiMixin from '../../mixins/apiMixin';
+import GroupState from '../../mixins/groupState';
+
+import {t} from '../../locale';
+
+const SuggestedOwners = React.createClass({
+  propTypes: {
+    event: React.PropTypes.object,
+  },
+
+  mixins: [
+    ApiMixin,
+    GroupState,
+    TooltipMixin({
+      html: true,
+      selector: '.tip',
+      container: 'body'
+    })
+  ],
+
+  getInitialState(){
+      return {owners: undefined};
+  },
+
+  componentDidMount(){
+    this.fetchData(this.props.event);
+  },
+
+  componentWillReceiveProps(nextProps){
+    this.fetchData(nextProps.event);
```

Comment:
You should check if the props even changed, otherwise I bet this triggers a few times per-page as-is (did you check the network graph)?